pvlib/shading.py

```diff
@@ -232,3 +232,76 @@ def sky_diffuse_passias(masking_angle):
        Available at https://www.nrel.gov/docs/fy18osti/67399.pdf
     """
     return 1 - cosd(masking_angle/2)**2
+
+
+def projected_solar_zenith_angle(axis_tilt, axis_azimuth,
+                                 solar_zenith, solar_azimuth):
```

Comment:
Can we swap the order around to `solar_zenith, solar_azimuth, axis_tilt, axis_azimuth` to match `pvlib.tracking.singleaxis`?